modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -4254,6 +4254,247 @@
 		"issueKey": "LPD-53535",
 		"to": "${variableName}.${prefixMethodName}CommerceInventoryBookedQuantity"
 	},
+	{
+		"classNames": [
+			"CommercePriceListService",
+			"CommercePriceListServiceUtil"
+		],
+		"from": "addCommercePriceList(long paramName, long paramName, long paramName, boolean paramName, long paramName, String paramName, double paramName, int paramName, int paramName, int paramName, int paramName, int paramName, int paramName, int paramName, int paramName, int paramName, int paramName, boolean paramName, ServiceContext paramName)",
+		"issueKey": "LPD-54343",
```

Comment:
THe issue key should be the task id not the subtask, could you change? @arodriguesrm